src/actions/lotesActions.js

```diff
@@ -0,0 +1,31 @@
+import { Firebase, FirebaseRef } from '../lib/firebase';
+
+/**
+ * Set an Error Message
+ */
+export function setError(message) {
+  return dispatch => new Promise(resolve => resolve(dispatch({
+    type: 'LOTES_ERROR',
+    data: message,
+  })));
+}
+
+/**
+ * Get Lotes
+ */
+export function getLotes(activeLotes) {
+  if (Firebase === null) {
+    return () => new Promise(resolve => resolve());
+  }
+
+  return dispatch => new Promise(resolve => FirebaseRef.child('lotes').orderByChild('title')
```

Comment:
I think this will need optimization in the future, since this requests ALL of the lotes and then we filter. There is no Firebase equivalent to SQL's SELECT * WHERE id IN (1,2,3). In Firebase you'll retrieve each item separately and idk how to yet. It's good for now but I'll rise and issue about it.